tests/unit/test_exporter.py

```diff
@@ -27,26 +27,30 @@ def test_parse_config(self, exporter_daemon):
     async def test_trigger(self, exporter_daemon):
         """Test trigger function."""
         statsd = exporter_daemon()
-        await statsd.trigger(once=True)
+
+        with mock.patch(
```

Comment:
nit: you could chain the with statements to save an indentation level

Reply:
Thanks for the suggestion